### PR TITLE
Header+logo comp implemented #4

### PR DESCRIPTION
**Pull request by yaasya21**
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React from "react";
-   
+import { Header } from './components/header/Header';
```
Comment (yaroslavUsenko) [inline src/App.js:2]: Change the folder name to "Header" and change the import to './components/Header'

File: src/components/header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Logo } from './logo/Logo';
```
Comment (yaroslavUsenko) [inline src/components/header/Header.jsx:2]: Change the folder name to "Logo" and change the import to './Logo'

**Review by yaroslavUsenko — COMMENTED** [review]
Please change the highlighted sections

File: src/App.js (hunk 1 above)
Comment (yaasya21) [reply src/App.js]: Done

File: src/components/Header/Header.module.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Nunito:wght@900&display=swap');
+
+.header {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0px 32px;
+
```
Comment (kirillserebrykov) [inline src/components/Header/Header.module.css:10]: delete whitespace

File: src/components/Header/Header.module.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 111px;
+    height: 28px;
+    border: 1px solid #FFFFFF;
+    border-radius: 400px;
+    flex: none;
+
```
Comment (kirillserebrykov) [inline src/components/Header/Header.module.css:38]: delete whitespace

Comment (yaasya21) [reply src/components/Header/Header.module.css]: Done

File: src/components/Header/Header.module.css (hunk 3 above)
Comment (yaasya21) [reply src/components/Header/Header.module.css]: Done

**Review by kirillserebrykov — APPROVED** [review]
Looks good to me

**Review by yaroslavUsenko — APPROVED** [review]
It was hard but you did it

**Review by VladislavZabilskij — APPROVED** [review]
All right.

**Pull request merged**